modules/tested-applications-for-glusterfs-storage.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/glusterfs-storage-optimization.doc
+
+[id='tested-applications-for-glusterfs-storage-{context}']
+= Tested applications for GlusterFS storage
+
+Red Hat performed extensive testing with these (no)SQL databases:
+
+-  Postgresql SQL v9.6
+-  MongoDB noSQL v3.2
```

Comment:
I need to check what version of Mongodb/Postgresql will be with OCP v4.0 templates